packages/tpetra/core/src/Tpetra_Details_FixedHashTable_def.hpp

```diff
@@ -76,7 +76,14 @@ namespace FHT {
 // we would need experiments to find out.
 template<class ExecSpace>
 bool worthBuildingFixedHashTableInParallel () {
-  return ExecSpace::concurrency() > 1;
+//  KDD 8/19:  A temporary fix for #5179.  It appears some errors come from
+//  KDD 8/19:  computeOffsetsFromCounts; this temporary patch avoids the most
+//  KDD 8/19:  reproducible of the errors.  We'll reverse this as we debug
+//  KDD 8/19:  the problem, but this temporary fix may allow users to make 
+//  KDD 8/19:  progress toward their milestones.
+//  return ExecSpace::concurrency() > 1;
+    return false;
```

Comment:
This makes me sad, but if it must be, it must be.

Reply:
@mhoemmen This is hopefully temporary.  The change buys us time to figure out the root.

Reply:
cause.

Reply:
Yup, intended to be temporary so we aren't a roadblock.

Reply:
Thanks for tracking this down!